user-access-commons/src/main/java/no/unit/nva/model/UserList.java

```diff
@@ -12,6 +12,7 @@
 
```

Comment:
github is stupid , it wont let me review non-changed diffs .. 

Anyhow, if this is an API object, maybe reflect in package name? :) ie `no.unit.nva.api.{requests,responses}.model` ? 
@axthosarouris

Reply:
Good point. Should we create a bug for dealing for both issues mentioned here?